src/core/actions/index.js

```diff
@@ -1,18 +1,23 @@
 import {
+  LOG_OUT_USER,
   SET_JWT,
   SET_CLIENT_APP,
   SET_LANG,
   ENTITIES_LOADED,
   SET_CURRENT_USER,
 } from 'core/constants';
 
-export function setJWT(token) {
+export function setJwt(token) {
```

Comment:
Yeah I'm cool with the revised format. I like `setJWT` more than `setJwt` but I like your example where `setJwtToken` is better and it's easier to grok the first rule so let's do that 👍